config/routes.rb

```diff
@@ -140,7 +140,8 @@
       end
     end
     resources :stats, only: :index
-    resource :news, path: 'news', only: [:show] do
+    resource :news, path: 'releases', only: [:show] do
+      get "/news" => redirect("/releases")
```

Comment:
If I'm reading this right, this will set up a redirect from `/releases/news/` to `/releases`, so I think line 144 and 143 need to swap, so the `get "/news"` line is outside the `resource :news` block?

Reply:
Alright, I'll make that correction.